### PR TITLE
Dashboard available from outside with API key

### PR DESCRIPTION
**Pull request by olivierteboul**

### CODE REVIEW

File: icubam/predicu/plot/operational_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
+
```
Comment (rth) [inline icubam/predicu/plot/operational_dashboard.py:1]: This should be put into a different folder, maybe ` icubam/predicu/dashboard`. All modules in ` icubam/predicu/plot/` are assumed to be plots with a specific API (e.g. have a plot function that returns something) and are automatically tested by pytest.

Comment (dulacarnold) [reply icubam/predicu/plot/operational_dashboard.py]: Ah good point!

**Comment by rth** [comment]
Nice work! Just one comment above otherwise LGTM (after a superficial review). 

 pre-commit (https://github.com/icubam/icubam/blob/master/doc/contributing.md#code-style) does help for dealing with linters :)

File: icubam/predicu/plot/operational_dashboard.py (hunk 1 above)
Comment (olivierteboul) [reply icubam/predicu/plot/operational_dashboard.py]: cool! done

**Comment by olivierteboul** [comment]
> Nice work! Just one comment above otherwise LGTM (after a superficial review).
> 
> pre-commit (https://github.com/icubam/icubam/blob/master/doc/contributing.md#code-style) does help for dealing with linters :)

thanks! Gabriel also pointed it to me. better indeed

**Pull request merged**